IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcGeoScienceFeatureType/Documentation.md

```diff
@@ -0,0 +1,2 @@
+The _IfcGeoScienceFeatureType_ provides the type information for _IfcGeoScienceFeature_ occurrences.
+An _IfcGeoScienceFeature_ represents a geological or geotechnical feature as an interpretation of factual data such as observations, measurements and tests.
```

Comment:
Do you want the description of IfcGeoScienceFeature on IfcGeoScienceFeatureType?

Reply:
Maybe not. I just looked at IfcPavementType as a template.